examples/app-with-rbac/src/README.md

```diff
@@ -0,0 +1,51 @@
+<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin. Uncomment and replace the content here before publishing.
```

Comment:
you can probably delete this README

Reply:
I'm not sure we should delete it. Developers use these examples as starters and without this README a plugin submission would likely be rejected during review.

Reply:
Like jackw, I'm also not sure this should be deleted. I'm also not sure it shouldn't be deleted, because I am not certain how these folders are built in the final plugin if certain files aren't in place, and so andresmgot may have a point. 

But what I do know is this:

1. The "root" folder is the README directly under app-with-rbac does not need to be built with the plugin. The one under src/ should exist, and I think it should have placeholder content because otherwise the plugin developers will not know what this folder is for, without guessing.

2. The README at the "root" folder needs an editorial pass and make a proposed commit. I will do this as soon as possible this week (I've been on PTO and just saw this PR today).

Reply:
Got it, just to clarify, the inner `README` does not need to exist in general but seem like we are leaving it as a placeholder for other examples too so sorry for the confusing instructions @gamab !

Reply:
No problem, thanks folks :pray: